sql.go

```diff
@@ -302,10 +298,6 @@ func (c wrappedConn) Query(query string, args []driver.Value) (driver.Rows, erro
 func (c wrappedConn) QueryContext(ctx context.Context, query string, args []driver.NamedValue) (rows driver.Rows, err error) {
 	span := c.GetSpan(ctx).NewChild(OpSQLConnQuery)
 	span.SetLabel("component", "database/sql")
-	span.SetLabel("query", query)
-	if !c.OmitArgs {
-		span.SetLabel("args", formatArgs(args))
-	}
```

Comment:
So...this was previously good! We don't want to remove this check as far as I understand. Instead we want to do this in other places which are adding the `args` tag such as `ExecContext` and `QueryContext`.

Reply:
That's odd ... the call to `logQuery` a bit later is supposed to take care of that, I'll look into why that is broken